dashboard/pages/relation_graph.tsx

```diff
@@ -152,40 +131,34 @@ export default function StreamingGraph() {
         }
       )
     }
-    refresh()
-    const interval = setInterval(refresh, INTERVAL_MS)
+    refresh() // run once immediately
+    const interval = setInterval(refresh, INTERVAL_MS) // and then run every interval
     return () => {
       clearInterval(interval)
     }
   }, [toast, resetEmbeddedBackPressures])
 
-  // Get relationId-relationId -> backpressure rate map
+  // Convert fragment-level backpressure rate map to relation-level backpressure rate
```

Comment:
QQ: what is relation-level backpressure rate?

Reply:
E.g. backpressure between MVs

Reply:
https://github.com/risingwavelabs/risingwave/pull/18280